plugins/source/aws/resources/services/lambda/runtimes_fetch.go

```diff
@@ -9,12 +9,8 @@ import (
 )
 
 func fetchLambdaRuntimes(_ context.Context, _ schema.ClientMeta, _ *schema.Resource, res chan<- any) error {
-	runtimes := make([]models.RuntimeWrapper, len(types.RuntimeProvidedal2.Values()))
-	for i, runtime := range types.RuntimeProvidedal2.Values() {
```

Comment:
Why are we no longer using `RuntimeProvidedal2`?

Reply:
Moved to `typed.Runtime("")`